toolchain/driver/compile_subcommand.cpp

```diff
@@ -176,6 +176,16 @@ Dump the tokens to stdout when lexed.
 )""",
       },
       [&](auto& arg_b) { arg_b.Set(&dump_tokens); });
+
+  b.AddFlag(
+      {
+          .name = "omit-file-boundary-tokens",
```

Comment:
Any reason to keep the flag? I'd be happy at least making the omitted the default, or even removing the flag entirely.

Reply:
So that people get the actual representation by default.